components/cmake/build_model.cmake

```diff
@@ -276,17 +276,17 @@ function(build_model COMP_CLASS COMP_NAME)
       if (USE_RRTMGPXX)
         target_link_libraries(${TARGET_NAME} PRIVATE rrtmgp rrtmgp_interface)
       endif()
-      if (USE_CLDERA_PROFILING)
-        if (DEBUG)
-          set (CLDERA_PATH_POSTFIX debug)
-        else()
-          set (CLDERA_PATH_POSTFIX release)
-        endif()
-        message ("CLDERA root DIR: ${CLDERA_PATH}/${CLDERA_PATH_POSTFIX}")
-        find_package(Cldera REQUIRED PATHS ${CLDERA_PATH}/${CLDERA_PATH_POSTFIX})
-        target_link_libraries(${TARGET_NAME} PRIVATE cldera-profiling)
-        target_compile_definitions(${TARGET_NAME} PRIVATE CLDERA_PROFILING)
+    endif()
+    if (USE_CLDERA_PROFILING)
```

Comment:
USE_CLDERA_PROFILING is still only defined based on CAM_CONFIG_OPTS. This mod is fine, but you also need to add logic to make USE_CLDERA_PROFILING depend on ELM configuration. Unless you will _always_ have cldera enabled in EAM whenever you have it enabled in ELM...